packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -902,6 +902,9 @@ export class GalleryContainer extends React.Component {
         onMouseLeave={this.onMouseLeave}
         ref={(e) => (this.galleryContainerRef = e)}
         tabIndex={-1}
+        style={{
```

Comment:
because of CSS property called `float`. if there is not enough room for the element to "float" to the side of the container it will drop to the bottom